Podfile

```diff
@@ -1,12 +1,10 @@
 source 'https://github.com/CocoaPods/Specs.git'
 
-platform :ios, '8.0'
+# platform :ios, '9.0'
 use_frameworks!
 
 target 'KitchenSink' do
-
-  pod 'SparkSDK'
-  pod 'Toast-Swift', '~> 1.1.0'
-  pod 'Cosmos', '~> 1.2'
-
+	pod 'SparkSDK', :git => 'https://github.com/ciscospark/spark-ios-sdk.git', :branch => 'swift3'
+	pod 'Cosmos', '~> 6.0.0'
+	pod 'Toast-Swift', git: 'https://github.com/scalessec/Toast-Swift.git', :tag => '2.0.0'
```

Comment:
At the time we made the pull request Toast-Swift didn't have their release pushed to CocoaPods yet, but had the tag. They have the release out now, so we can change that.